scripts/us_bea/states_gdp/import_industry_data_and_gen_mcf.py

```diff
@@ -20,148 +20,143 @@
     python3 import_data.py
 """
 from urllib.request import urlopen
-from absl import app
-import pandas as pd
 import io
 import zipfile
 import csv
 import re
+from absl import app
+import pandas as pd
+from import_data import StateGDPDataLoader
```

Comment:
https://google.github.io/styleguide/pyguide.html#22-imports.  Is "StateGDPDataLoader" a class?  If so, we should use "import import_data" and replace all instances with "import_data.StateGDPDataLoader".

Reply:
Done.